### PR TITLE
created transperency post

### PR DESCRIPTION
**Pull request by Ekwav**

### CODE REVIEW

File: src/pages/transparancy.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Code in the open
+Pretty much all our code is or will be published under the `AGPL-3` licence. 
+This means it is free to use, copy, modify and redistribute in any form, as long as you also publish any modifications you made (with the same licence).  
+Effectively:
+> you can take our code if you also allow us to take yours. Win-Win.
+
+
+## What happens when
+### competitor copies and uses our code
```
Comment (Flou21) [inline src/pages/transparancy.md:35]: competitors?
```suggestion
### competitors copies and uses our code
```

File: src/pages/transparancy.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Speaking of partnering
+If you are or want to develop something but lack the expertise to publish it feel free to contact us.
+We were there as well and it would have been really great to get someone to help.
+The only catch being that when we help you to reach success we want to have a cut.
+If it doesn't work out, sad, but whatever.
+Please note that this is just a general offer to partner, we might not have time to help you to the extend you hope for.
+We are not teaching you how to code 101. We want to **help you build something amazing**. 
+
+## Colaborating with us
```
Comment (matthias-luger) [inline src/pages/transparancy.md:56]: Remove this paragraph

File: src/pages/transparancy.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+This means it is free to use, copy, modify and redistribute in any form, as long as you also publish any modifications you made (with the same licence).  
+Effectively:
+> you can take our code if you also allow us to take yours. Win-Win.
+
+
+## What happens when
+### competitors copies and uses our code
+What about that? That basically proves our system is better than his. Which is a fantastic argument when getting new customers.  
+
+### he sells the code
```
Comment (matthias-luger) [inline src/pages/transparancy.md:38]: Capital "H"

File: src/pages/transparancy.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Code in the open
+Pretty much all our code is or will be published under the `AGPL-3` licence. 
+This means it is free to use, copy, modify and redistribute in any form, as long as you also publish any modifications you made (with the same licence).  
+Effectively:
+> you can take our code if you also allow us to take yours. Win-Win.
+
+
+## What happens when
+### competitors copies and uses our code
```
Comment (matthias-luger) [inline src/pages/transparancy.md:35]: Capital "C"

**Review by matthias-luger — CHANGES_REQUESTED** [review]

File: src/pages/transparancy.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+We are not teaching you how to code 101. We want to **help you build something amazing**. 
+
+## Colaborating with us
+We agreed on sharing our revenue as a 50/50 cut between the company and all contributors of at least one active feature.
+Active in that case meaning that is still in the source code when revenue is generated.
+Because `feature` is a vague term here are some guidelines. A feature is:
+* something you would put into a class
+
+To reduce bugs, 5% of the company split is reserved for bug bounty. 
+If there are no bugs reported for 12 months the bounty for the first 3 months is treated as normal revenue for collaborators
```
Comment (matthias-luger) [inline src/pages/transparancy.md]: Delete
```suggestion
```

File: src/pages/transparancy.md (hunk 4 above)
Comment (matthias-luger) [inline src/pages/transparancy.md]:
```suggestion
### Competitor copies and uses our code
```

File: src/pages/transparancy.md (hunk 3 above)
Comment (matthias-luger) [inline src/pages/transparancy.md]:
```suggestion
### He sells the code 
```

**Review by matthias-luger — CHANGES_REQUESTED** [review]

File: src/pages/transparancy.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+title: "Commiting to transparancy"
+description: 'We pledge to be as transparent and open as possible'
+date: 2021-08-02
+---
+
+Some mission statements that we will refer to going forward. 
+
+## Working in the open
+We intend to be 100% transparent.   
+This is actually really easy when using github to colaborate. In fact this [blog](https://github.com/Coflnet/Blog) is a repository on github as well.
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
This is actually really easy when using github to collaborate. In fact this [blog](https://github.com/Coflnet/Blog) is a repository on github as well. 
```

File: src/pages/transparancy.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+No matter if you are just bored or like to correct others. You are free to browse through anything we built.
+
+### Cons:
+#### Everyone can see our mistakes
+While I myself were very afraid of this at first it turns out that this is a Pro as well. 
+The ultimate target is to produce good code. And unlike with cookie many coders usually produce better code.
+
+
+## Code in the open
+Pretty much all our code is or will be published under the `AGPL-3` licence.
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
Pretty much all our code is or will be published under the `AGPL-3` license. 
```

File: src/pages/transparancy.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+### Cons:
+#### Everyone can see our mistakes
+While I myself were very afraid of this at first it turns out that this is a Pro as well. 
+The ultimate target is to produce good code. And unlike with cookie many coders usually produce better code.
+
+
+## Code in the open
+Pretty much all our code is or will be published under the `AGPL-3` licence. 
+This means it is free to use, copy, modify and redistribute in any form, as long as you also publish any modifications you made (with the same licence).
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
This means it is free to use, copy, modify and redistribute in any form, as long as you also publish any modifications you made (with the same license).  
```

File: src/pages/transparancy.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+Effectively:
+> you can take our code if you also allow us to take yours. Win-Win.
+
+
+## What happens when
+### Competitor copies and uses our code
+What about that? That basically proves our system is better than his. Which is a fantastic argument when getting new customers.  
+
+### He sells the code 
+Nothing wrong with that either, as long as he tells the buyer that it is licenced under the `AGPL-3`.
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
Nothing wrong with that either, as long as he tells the buyer that it is licensed under the `AGPL-3`.
```

File: src/pages/transparancy.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+That will also require him to mention that we wrote it.  
+
+### We get bought
+Not a lot changes either. The licencensing of the existing code is still `AGPL-3`.
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
Not a lot changes either. The licensing of the existing code is still `AGPL-3`.
```

File: src/pages/transparancy.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+With audacity recently being bought up and a questionable licencing agreement being made we put some consideration into this.
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
With audacity recently being bought up and a questionable licensing agreement being made we put some consideration into this.
```

File: src/pages/transparancy.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+We belive in retaining ownership and partnering with others to accomplish something else is the better way to go.
+Both parties are motivated to accomplish a common goal and not to screw each other over in a licencing agreement.
```
Comment (Flou21) [inline src/pages/transparancy.md]:
```suggestion
Both parties are motivated to accomplish a common goal and not to screw each other over in a licensing agreement.
```

**Review by Flou21 — CHANGES_REQUESTED** [review]

**Review by Flou21 — APPROVED** [review]
lgtm

File: src/pages/transparancy.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Commiting to transparancy"
+description: 'We pledge to be as transparent and open as possible'
+date: 2021-08-02
```
Comment (matthias-luger) [inline src/pages/transparancy.md:4]: update date

Comment (matthias-luger) [inline src/pages/transparancy.md]:
```suggestion
date: 2021-08-06
```

Comment (Ekwav) [reply src/pages/transparancy.md]: -_-

**Pull request merged**